### PR TITLE
all tests pass on osx!

### PR DESCRIPTION
**Pull request by morethanreal**
@cdata @notwaldorf PTAL

### CODE REVIEW

**Comment by cdata** [comment]
> all tests pass in OSX
Otherwise known as all tests pass in Safari and Chrome? :)

File: paper-input-container.html [hunk 1]
```diff
@@ -336,8 +344,7 @@
       }
 
       // notify add-ons
-      var addons = Polymer.dom(this.$.addOnContent).getDistributedNodes();
-      for (var a, i = 0; a = addons[i]; i++) {
+      for (var a, i = 0; a = this._addons[i]; i++) {
```
Comment (cdata) [inline paper-input-container.html]: I say meh to these abbreviations. They think they are so cool. What do they even mean?! :unamused:

File: test/paper-input.html [hunk 2]
```diff
@@ -49,6 +49,17 @@
 
     suite('basic', function() {
 
+      function forceXIfStamp(node) {
+        var templates = Polymer.dom(node.root).querySelectorAll('template[is=x-if]');
+        for (var t, i = 0; t = templates[i]; i++) {
```
Comment (cdata) [inline test/paper-input.html]: Ditto re: https://github.com/PolymerElements/paper-input/pull/7/files#r29080403

File: test/paper-input.html [hunk 3]
```diff
@@ -49,6 +49,17 @@
 
     suite('basic', function() {
 
+      function forceXIfStamp(node) {
```
Comment (cdata) [inline test/paper-input.html]: This looks like it might make a great iron-test-helper :)

**Comment by cdata** [comment]
Small nits, otherwise LGTM :boat:

**Comment by morethanreal** [comment]
Also passing in firefox locally, not sure why it fails in the cloud :(

File: test/paper-input.html (hunk 3 above)
Comment (morethanreal) [inline test/paper-input.html]: https://github.com/PolymerElements/iron-test-helpers/pull/3

File: paper-input-container.html (hunk 1 above)
Comment (morethanreal) [inline paper-input-container.html]: I can change `a`, but I think using `i` for the index is pretty standard.

**Pull request merged**